addons/bus/static/src/services/bus_service.js

```diff
@@ -245,7 +245,7 @@ export const busService = {
             subscribe(notificationType, callback) {
                 const wrapper = ({ detail }) => {
                     const { id, payload } = detail;
-                    callback(payload, { id });
+                    callback(JSON.parse(JSON.stringify(payload)), { id });
```

Comment:
Note: a notif handler could mutate payload in place, leading to other handlers not receiving the expected payload